proto/agent.proto

```diff
@@ -363,12 +363,12 @@ message Connection {
 }
 
 message Addr {
-	// post-resolution field; deprecated in favor of hostId
-	Host host = 1  [deprecated = true];
+	// 1 is deprecated
```

Comment:
thoughts on using `reserved` instead of relying on these comments? 

https://developers.google.com/protocol-buffers/docs/proto#reserved

Reply:
Nice - TIL.

Switched over all of our deprecated comments to this field.